src/layouts/DefaultLayout.tsx

```diff
@@ -24,7 +26,7 @@ export default function DefaultLayout({ children }: ChildrenComponent) {
             component="div"
             sx={{
               flexGrow: 1,
-              backgroundColor: '#e6e6e6'
+              backgroundColor: mode === 'light' ? '#e6e6e6' : '#606060'
```

Comment:
lets use the `useTheme` hook and use the color, if we dont have it just create a new color inside the themes
```
...lightThemeColros,
colorName: {
    main: '#e6e6e6'
}
```

Reply:
I discovered the power of useTheme. In this object have property in pallete grey and info for mode and i used them.
 backgroundColor:
                theme.palette.mode === 'light'
                  ? theme.palette.grey['300']
                  : theme.palette.grey['800']